docs/examples/index.maplibre.opacity.3857.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Adjust a layer's opacity</title>
+        <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+        <script
+            src="https://unpkg.com/maplibre-gl@1.14.0-rc.1/dist/maplibre-gl.js"
```

Comment:
This verision is super out of date, I also wonder if we should just point at maplibre examples if they exist instead of re-inventing them here?